recipe/patches/0004-Specify-PATH_SUFFIX-for-rxspencer-include-dirs.patch

```diff
@@ -1,22 +0,0 @@
-From a04e2df79dd9859cdac20dc36205c32803154ade Mon Sep 17 00:00:00 2001
-From: Nehal J Wani <nehaljw.kkd1@gmail.com>
```

Comment:
Appears the file this patch applies to no longer exists. So am trying just dropping the patch

Here's the output from CI below with log linked after

```
Applying patch: /home/conda/recipe_root/patches/0004-Specify-PATH_SUFFIX-for-rxspencer-include-dirs.patch
Patch level ambiguous, selecting least deep
Applying patch: /home/conda/recipe_root/patches/0004-Specify-PATH_SUFFIX-for-rxspencer-include-dirs.patch with args:
['-Np0', '-i', '/tmp/tmpy7cncc1y/0004-Specify-PATH_SUFFIX-for-rxspencer-include-dirs.patch.native', '--binary']
can't find file to patch at input line 14
Perhaps you used the wrong -p or --strip option?
```

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=290782&view=logs&jobId=656edd35-690f-5c53-9ba3-09c10d0bea97&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d

cc @nehaljwani